book/website/collaboration/inclusive-events/inclusive-events-location.md

```diff
@@ -0,0 +1,249 @@
+(cl-inclusive-events-location)=
+# Location
+
+The location of your event plays a pivotal role in supporting or hindering how accessible and inclusive your event will be for people with lived experience.
+
+This section provides guidance that is most relevant for in-person or hybrid events. 
+
+Whether it's a conference, seminar, or any gathering, the venue's physical location, its layout, and surrounding infrastructure can significantly 
+influence the experience of attendees with lived experience, especially those with disabilities
+
+Identifying and addressing the access needs of your attendees will help to both foster a welcoming environment and ensure that all attendees are able to fully engage in your event.
+
+(cl-inclusive-events-choosing-venue)=
+## Choosing a Venue
```

Comment:
in terms of a list?

I can add a list into this section, or add it to the checklist page.

Reply:
Sorry, that was a confusing comment! And yeah, I think it meant to signpost to the checklist page which would be helpful to have on the day of doing the venue walkthrough! 🙂

Reply:
Sounds like a plan. I'll add in a place holder and leave it in until the checklist page is done. I'll also make a note as it will probably be something that is a final check/addition along with the toc file and references.